app/javascript/Components/Helpers/table_helpers.jsx

```diff
@@ -121,11 +121,13 @@ export function selectFilter({filter, onChange, column}) {
 
 export function markingStateColumn(marking_states, markingStateFilter, ...override_keys) {
   return {
-    Header: I18n.t("activerecord.attributes.result.marking_state"),
-    accessor: "marking_state",
-    Cell: row => {
+    header: I18n.t("activerecord.attributes.result.marking_state"),
```

Comment:
Oh right, overall you should not change this function (since it's also used in the `SubmissionTable` component, which is on an older version of React table). Instead, you can make a copy of the function and move it directly into the `assignment_summary` file.